documentation/API-Testing-Guidelines.md

```diff
@@ -269,7 +269,7 @@ Typical response validations:
 - the response body complies with the OAS schema at "{oas_schema_json_path}"
 ```
 
-- additional constraints for optional or dependant properties in the response body, e.g.:
+- additional constraints for optional or independent properties in the response body, e.g.:
```

Comment:
```suggestion
- additional constraints for optional or dependent properties in the response body, e.g.:
```

Reply:
It is dependant due to the use cases illustrated (clause "if"):

- the response property "{json_path}" only exists **if** property "{json_path}" exists
- the response property "{json_path}" only exists **if** property "{json_path}" is "{value}"